libs/wire-api/src/Wire/API/Team/Feature.hs

```diff
@@ -451,6 +456,82 @@ defaultSelfDeletingMessagesStatus =
     TeamFeatureEnabled
     (TeamFeatureSelfDeletingMessagesConfig 0)
 
+----------------------------------------------------------------------
+-- PaymentStatus
+
+data PaymentStatusUpdate = Lock | Unlock
+  deriving stock (Eq, Show, Generic)
+
+instance FromHttpApiData PaymentStatusUpdate where
+  parseUrlPiece = \case
+    "unlock" -> return Unlock
+    "lock" -> return Lock
+    x -> Left $ "Invalid path: " <> x
+
+data PaymentStatusValue = PaymentLocked | PaymentUnlocked
+  deriving stock (Eq, Show, Generic)
+  deriving (Arbitrary) via (GenericUniform PaymentStatusValue)
+  deriving (ToJSON, FromJSON, S.ToSchema) via (Schema PaymentStatusValue)
+
+newtype PaymentStatus = PaymentStatus
+  { paymentStatus :: PaymentStatusValue
+  }
+  deriving stock (Eq, Show, Generic)
+  deriving (FromJSON, ToJSON, S.ToSchema) via (Schema PaymentStatus)
+  deriving (Arbitrary) via (GenericUniform PaymentStatus)
+
+instance ToSchema PaymentStatus where
+  schema =
+    object "PaymentStatus" $
+      PaymentStatus
+        <$> paymentStatus .= field "paymentStatus" schema
+
+modelPaymentStatus :: Doc.Model
+modelPaymentStatus =
+  Doc.defineModel "PaymentStatus" $ do
+    Doc.property "paymentStatus" typePaymentStatusValue $ Doc.description "payment status: locked | unlocked"
```

Comment:
```suggestion
    Doc.property "paymentStatus" typePaymentStatusValue
```

the description doesn't add any information to `typePaymentStatusValue`, no?

Reply:
That is true. I'll leave it empty then.